v5/getting-started/linux.rst

```diff
@@ -5,17 +5,17 @@ Installing on Linux
 Installing the toolchain
 ------------------------
 
-1. Follow the instructions found `here <https://launchpad.net/~team-gcc-arm-embedded/+archive/ubuntu/ppa>`_ to add and install the latest version of the GNU Arm Embedded toolchain.
+1. If you are using a Debian-based distribution of Linux of version >18.04, you can find the toolchain `here <https://www.ubuntuupdates.org/package/core/disco/universe/base/gcc-arm-none-eabi>`_.

```

Comment:
Is this true? I haven't heard anything about this. The version 18.04 here implies that you mean this works specifically on Ubuntu, rather than on any compatible Debian-derived distribution, so I'd like to verify that it works on those too.

Reply:
I've actually only dealt with ubuntu deviations while installing pros i could try it on Debian and see if there are any errors with that.